### DISCUSSION TITLE
육각형 개발자

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#475] [4. 기타 글 모음]
https://shkisme.vercel.app/hexagon-developer

육각형 개발자 세미나 내용 정리 글입니당

여기 계시는 분들은 다 육각형 개발자들이신 듯... 👍

**Comment by 02ggang9** [comment]
글 내용 재미있게 잘 읽었습니다! 저도 세미나 내용 들어봤으면 .. ㅠ
